tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/HashBasedDeduplicationTests.cs

```diff
@@ -15,7 +15,7 @@ public void GivenTheSameVulnerabilityInstance_WhenAddedToDeduplication_OnlyOneIs
     {
         var instance = new HashBasedDeduplication();
         var ranges1 = new Range[] { new Range(1, 1, new Source(0, "sourceName", "sourceValue")) };
-        var vulnerability = new Vulnerability(VulnerabilityTypeName.WeakHash, new Location("path.cs", 23), new Evidence("MD5", ranges1));
```

Comment:
FYI, `0` is an invalid span ID. You should maybe check for this in the `Location` constructor and set it to `null`. Or just use `null` in these tests ;)

Reply:
It makes sense setting the id to null if it is 0. Thanks for the tip!